### ISSUE TITLE
xawtv-3.103 fails to build

### ISSUE THREAD

**Issue by kardasa** [jorgicio/jorgicio-gentoo#82]
xawtv buid fails with:
libng/contrib-plugins/linear-blend.c: Assembler messages:
libng/contrib-plugins/linear-blend.c:43: Error: `(%rcx,%r8d)' is not a valid base/index expression
libng/contrib-plugins/linear-blend.c:55: Error: `(%rcx,%r8d,4)' is not a valid base/index expression
libng/contrib-plugins/linear-blend.c:66: Error: `(%rcx,%r8d,4)' is not a valid base/index expression
libng/contrib-plugins/linear-blend.c:71: Error: `(%rcx,%r8d,8)' is not a valid base/index expression
make: *** [mk/Compile.mk:54: libng/contrib-plugins/linear-blend.o] Error 1
Full build log:
https://gist.github.com/kardasa/1dc9c14a096a68ec6ed2b107c91435f2

**Comment by jorgicio** [comment]
Hi! Here xawtv builds fine, using amd64 architecture. Please, attach your `emerge --info` and the gcc version you're using. I'm using the stable one (4.9.4)

**Comment by kardasa** [comment]
Thanks for response. I was trying to build using both stable 4.9.4 and 5.4.0 outcome is the same.
My emerge --info:
https://gist.github.com/kardasa/b634e6ccc27da3e7cf0da177bbaa3b3b

**Comment by jorgicio** [comment]
I got what happens: if fails when I have the mmx enabled. The same happened to me until I disabled that useflag.
I tried to masked from the overlay but nothing happens and I don't know why.
A workaround:
add this line to your `/etc/portage/package.use/use`
`media-tv/xawtv -cpu_flags_x86_mmx`
Then resync my overlay (I added another patch, unrelated to this issue), and reinstall the package:
`emerge -1 xawtv`
It would build fine.
